src/projects/EnsureThat/Enforcers/EnumArg.cs

```diff
@@ -24,7 +26,7 @@ public sealed class EnumArg
         /// Abc.A | Abc.B IsDefined=true (due to Abc.AB)
         /// Abc.A | Abc.C IsDefined=false (A and C are both valid, the composite is valid due to `Flags` attribute, but the composite is not a named enum value
         /// </example>
-        public T IsDefined<T>(T value, [InvokerParameterName] string paramName = null, OptsFn optsFn = null) where T : struct, Enum
+        public T IsStrictlyDefined<T>(T value, [InvokerParameterName] string paramName = null, OptsFn optsFn = null) where T : struct, Enum
```

Comment:
I would match parity of names - IsDefined should work like Enum.IsDefined, IMO

Reply:
What about flags then? `IsAnyDefined`?

Reply:
I believe the `IsDefined` name should map purely to BCL `Enum.IsDefined`.

The member supporting flags should then be `IsDefinedExtended`. I know it's a bit weird naming perhaps. But at least they will be discover-able together and one can use the intellisense docs to understand what is extended.